data/json/mutations/mutations.json

```diff
@@ -6158,7 +6161,7 @@
     "id": "THRESH_PLANT",
     "name": { "str": "Plant" },
     "points": 1,
-    "description": "Well, you still have those other walking flowers-and the mushrooms too-to deal with.  But you'll manage.",
+    "description": "Well, you still have those other walking flowers--and the mushrooms, too--to deal with.  But you'll manage.",
```

Comment:
```suggestion
    "description": "Well, you still have those other walking flowers - and the mushrooms, too - to deal with.  But you'll manage.",
```

Reply:
Convention in this file and others seems to be two dashes, unspaced. See lines 3232, 4892, 4931, &c. If we want to propose a full rewrite of documentation, we can do that, but I think it's best to stick to convention.